.goreleaser.yml

```diff
@@ -36,17 +36,22 @@ brews:
       name: devx-sauce-bot
       email: devx.bot@saucelabs.com
 archives:
-  - replacements:
-      darwin: mac
-      linux: linux
-      windows: win
-      amd64: 64-bit
-      386: 32-bit
+  - name_template: >-
+      {{ .ProjectName }}_
+      {{ .Version }}_
+      {{- if eq .Os "darwin"}}mac_
+      {{- else if eq .Os "linux" }}linux_
+      {{- else if eq .Os "windows" }}win_
+      {{- else }}{{ .Os }}{{ end }}_
+      {{- if eq .Arch "amd64" }}64-bit
+      {{- else if eq .Arch "386" }}32-bit
+      {{- else }}{{ .Arch }}{{ end }}
     format_overrides:
       - goos: windows
         format: zip
     files:
       - LICENSE
+    rlcp: true
```

Comment:
Maybe we should have this in another PR ?

Reply:
I've put it in a dedicated PR. 
Did two small changes, one due to a typo, second due to what is likely to be a bug in `goreleaser`.

`      {{- else }}{{ .Os }}{{ end }}_` => `      {{- else }}{{ .Os }}_{{ end }}`

https://github.com/saucelabs/saucectl/pull/645